lib/deploy/migration.rb

```diff
@@ -0,0 +1,13 @@
+module Deploy
+  module Migration
+    def connection
+      connection = super
+      new_statement_timeout = ENV['MIGRATION_STATEMENT_TIMEOUT']
+      if new_statement_timeout && !@migration_statement_timeout_set
+        connection.execute("SET statement_timeout = #{new_statement_timeout.to_i}")
+        @migration_statement_timeout_set = true
+      end
+      connection
+    end
+  end
+end
```

Comment:
I'm not sure how to test this. Alternatively, to avoid Code Climate from complaining, we can use the `slowpoke` gem instead (which is where I got this code from), and only require this module.

Reply:
Unfortunately, that won't work because the gem depends on a method in the main class that we don't want to require. So, our options are to tell Code Climate to ignore this file for test coverage or to build our own gem. I will try the former.

Reply:
👍